app/frontend/wallet/helpers/CachedDeriveXpubFactory.ts

```diff
@@ -53,25 +53,28 @@ function CachedDeriveXpubFactory(derivationScheme, shouldExportPubKeyBulk, deriv
     const accountIndex = derivationPath[2] - HARDENED_THRESHOLD
     const currentAccountPage = Math.floor(accountIndex / MAX_BULK_EXPORT_AMOUNT)
 
-    for (let i = 0; i < MAX_BULK_EXPORT_AMOUNT; i += 1) {
+    // we want to scale the number of exported keys up to 3 times the MAX_BULK_EXPORT_AMOUNT
+    const nAccounts = Math.min(currentAccountPage + 1, 3) * MAX_BULK_EXPORT_AMOUNT
```

Comment:
This is getting really misleading - MAX_BULK_EXPORT_AMOUNT is not the maximum number of exported public keys anymore and the "current page" is not really the "current page" as these are now of variable size and not really "pages" as units of exported data.

I'd consider renaming the `MAX_BULK_EXPORT_AMOUNT` constant to `EXPORTED_PATHS_SLICE_SIZE`, `currentAccountPage` to `currentAccountSlice`, and to get the `nAccounts` I'd rather make a helper function `getExportedBulkSize` which would take as an argument the currentAccountSlice and have explicit `if-else` for returning the bulk size than a rather cryptic, albeit short formula